server/app.ts

```diff
@@ -149,6 +147,8 @@ export default function createApp(services: Services): Express {
     app.use('/assets/images/icons', express.static(path.join(process.cwd(), dir), cacheControl))
   })
 
+  app.use(bunyanRequestLogger({ name: 'Use of force http', serializers: loggingSerialiser }).requestLogger())
```

Comment:
moving this down so we don't log every request for assets as `trace`, will still do as `requests` though